packages/website/pages/stats.js

```diff
@@ -109,30 +109,97 @@ export default function Stats({ logos }) {
     setStatsLoading(false)
   }
 
-  const Marquee = () => {
+  const DescriptionSection = () => {
+    const pClass = 'my-8 text-xl leading-8'
     return (
-      <div className="relative w-screen max-w-100 h-[100px] border-y border-black flex items-center justify-center">
-        <p className="chicagoflf p-4 m-0 text-[clamp(16px,_2.6rem,_6vw)]">
-          NFT.Storage is storing...
-        </p>
+      <div className="max-w-7xl mx-auto py-16 px-6 sm:px-16 flex flex-col lg:flex-row">
+        <div className="lg:w-1/2 pr-2 leading">
+          <h1 className="text-4xl md:text-5xl chicagoflf">
+            The Case for{' '}
+            <span className="sm:whitespace-nowrap">Decentralized Storage</span>
+          </h1>
+          <div>
+            <p className={pClass}>
+              NFTs represent a new era in verifiable and trustless ownership,
+              but what if that awesome 1/1 art piece you bought wasn’t using a
+              decentralized persistent storage solution?
+            </p>
+            <p className={pClass}>
+              Well...the proof of your purchase doesn’t mean as much now does it
+              if the art can simply be changed or moved somewhere else?
+            </p>
+            <p className={pClass}>
+              Of course NFTs can be more than just art, but NFTs without
+              trustless storage fall into the same traps that web3 is aiming to
+              solve.
+            </p>
+          </div>
+        </div>
+        <div className="flex lg:w-1/2 items-center justify-center pl-2">
+          <Img
```

Comment:
@e-schneid for some reason, the `decentralized.png` image is showing as "not found" on the preview link. It is using the cloudflare image loader and my guess is that it may only be effecting the preview instance but I wanted to point this out.

Reply:
thanks @johnathonroach .  I'll do some research to make sure this is the case

Reply:
thanks for testing it out!  I'll do some research to make sure it's just the preview.